pages/cart.css

```diff
@@ -64,3 +68,54 @@
   color: gray;
   cursor: pointer;
 }
+/* card-total */
+.price-details-container {
+  background-color: #fdf4f3;
+
+  margin: 1rem;
+  padding: 1rem;
+}
+.bold {
+  font-weight: 600;
+  font-size: 1.2rem;
+}
+hr {
```

Comment:
try to reduce such short code by making utility classes. would be a good practice

Reply:
yes you are right